test/services/helpdesk/index.test.js

```diff
@@ -88,8 +88,7 @@ describe('helpdesk service', () => {
 		const postBody = {
 			type: 'contactHPI',
 			subject: 'Dies ist ein Titel 4',
```

Comment:
Add `[SC-TEST]` to the beginning of all test mail subjects, based on that all testers may now this can be ignored if they receive such an email. do not add this here but in the mail sender when `NODE_ENV` does not equal `production`

Reply:
done, even if I think that it should happen during the test, otherwise on, e.g staging, tests will not get the prefix and thus can not be gefiltert.